bench/Micro.Benchmark/Benchmarks/HopeEncoder.cs

```diff
@@ -105,6 +105,18 @@ public State(int size)
 
             public Span<byte> EncodingTable => new Span<byte>(_value).Slice(0, _value.Length / 2);
             public Span<byte> DecodingTable => new Span<byte>(_value).Slice(_value.Length / 2);
+
+            public bool CanGrow => false;
+
+            public void Dispose()
+            {
+                throw new NotImplementedException();
+            }
+
+            public void Grow(int minimumSize)
+            {
+                throw new NotImplementedException();
```

Comment:
Shouldn't we rather throw NotSupportedException? Same like you do in NativeMemoryEncoderState.cs

Reply:
It's never gonna be tested, it is there for triggering a failure at the user but it is a code sanity thing. Yes.